### PR TITLE
choose faction view and photos

### PR DESCRIPTION
**Pull request by brookermia**

### CODE REVIEW

File: src/components/ContentContainer.tsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
+import { Grid } from '@mui/material';
 import React, { FunctionComponent, ReactElement } from 'react';
-import Grid from '@mui/material/Unstable_Grid2';
+// import Grid from '@mui/material/Unstable_Grid2';
 import TopLogo from '../assets/images/top_logo.png';
 
 interface Props {
-  children: ReactElement[] | ReactElement;
+  children: ReactElement[] | ReactComponent[];
```
Comment (brookermia) [inline src/components/ContentContainer.tsx:7]: @fargusplumdoodle should this be ReactComponent[] or stay ReactElement?

Comment (fargusplumdoodle) [reply src/components/ContentContainer.tsx:7]: Yes this should probable be the way it was before, I don't know what putting it as a `ReactComponent` will do here

Comment (fargusplumdoodle) [reply src/components/ContentContainer.tsx:7]: Maybe stick with Grid 2 if it doesn't cause issues?

File: src/views/ChooseFactionView/ChooseFactionButton.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      variant="outlined"
+      sx={{
+        width: '100%',
+        height: '100%',
+        borderRadius: '4px',
+        padding: '0px',
+        ' :hover': {
+          backgroundImage: `url(${image})`,
+          // color: 'white',
+          // borderColor: 'white',
```
Comment (fargusplumdoodle) [inline src/views/ChooseFactionView/ChooseFactionButton.tsx:28]: We should remove this commented out code

File: src/views/ChooseFactionView/ChooseFactionList.ts [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+  {
+    id: 'Vikings',
+    name: 'Vikings',
+    img: VIKINGS,
+  },
+];
+
+export default {
+  FantasyFactions,
+  DarkAgesFactions,
```
Comment (fargusplumdoodle) [inline src/views/ChooseFactionView/ChooseFactionList.ts:105]: Minor thing that will matter later, we should have these keys correspond to the IDS for [fantasy](https://github.com/Tremors-Of-War/tremors-of-war/blob/4e60ae9e9af87ee56006b0bd77ef1ed078401aca/src/data.json#L673-L674) and [dark ages](https://github.com/Tremors-Of-War/tremors-of-war/blob/4e60ae9e9af87ee56006b0bd77ef1ed078401aca/src/data.json#L2747) in our database.

I realise those IDs are not in our existing naming scheme, but maybe we should change that in our `data.json`

File: src/views/ChooseFactionView/ChooseFactionView.tsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import React, { FunctionComponent } from 'react';
+import { Button, Grid, Typography } from '@mui/material';
+// import Grid from '@mui/material/Unstable_Grid2';
```
Comment (fargusplumdoodle) [inline src/views/ChooseFactionView/ChooseFactionView.tsx:3]: comment yo

**Review by fargusplumdoodle — APPROVED** [review]

**Pull request merged**